tasks/upgrade_debian.yml

```diff
@@ -48,4 +48,8 @@
     name: "{{ item }}"
     state: restarted
   with_items: "{{ services.stdout_lines }}"
-  when: services.stdout_lines and upgrade_restart_services and item not in upgrade_restart_services_ignore | union(['dbus'])
+  when: services.stdout_lines and upgrade_restart_services and item not in upgrade_restart_services_ignore | union(['dbus', 'systemd'])
+
+- name: Reexec systemd
+  command: systemctl daemon-reexec
+  when: "services.stdout_lines and 'systemd' in services.stdout_lines"
```

Comment:
`upgrade_restart_services_ignore` should be respected here, i.e. if I don't want systemd restarted at all.

Reply:
You don't restart systemd. You reexec the daemon

Reply:
Restarting systemd would result in a kernel panic

Reply:
Okay. *reexeced

Reply:
But why wouldn't one? I can understand someone wouldn't want to restart their databse because it would mean downtime. Or OVS (you know what I mean). But a daemon-reexec means aboslutely no downtime so there is no reason to not restart it

Reply:
I don't know, but it is not the expected behaviour when reading the readme.
Just document this exception.

Reply:
Pls rereview